mypy_django_plugin/transformers/orm_lookups.py

```diff
@@ -29,7 +30,11 @@ def typecheck_queryset_filter(ctx: MethodContext, django_context: DjangoContext)
         ):
             provided_type = resolve_combinable_type(provided_type, django_context)
 
-        lookup_type = django_context.resolve_lookup_expected_type(ctx, model_cls, lookup_kwarg)
+        lookup_type: MypyType
+        if is_annotated_model_fullname(model_cls_fullname):
+            lookup_type = AnyType(TypeOfAny.explicit)
```

Comment:
I prefer `.implementation_detail`, because `.explicit` can raise errors if `no-explicit-any` is set.